src/topics/tools.js

```diff
@@ -92,6 +92,24 @@ module.exports = function (Topics) {
         return await toggleLock(tid, uid, false);
     };
 
+    async function resolved(tid, uid) {
```

Comment:
nit: based on current naming conventions of this codebase, this should be `resolve` instead of `resolved`